server/channels/app/channel.go

```diff
@@ -1935,14 +1935,6 @@ func (a *App) GetChannelsForTeamForUser(c request.CTX, teamID string, userID str
 	return a.Srv().getChannelsForTeamForUser(c, teamID, userID, opts)
 }
 
-func (a *App) GetChannelsForTeamForUserWithCursor(c request.CTX, teamID string, userID string, opts *model.ChannelSearchOpts, afterChannelID string) (model.ChannelList, *model.AppError) {
```

Comment:
Do you think we're going to want these cursors in the future at all? I suppose we can always re-add them later on, but these feel like they should be useful even without GraphQL

Reply:
If we need them, we can always add them back later. But I don't want unused code lying around.